index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 
-**If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
+**[Get in touch](mailto:hi@tkiley.co.uk) today to learn how to use software industries best practices at your work.**
 
-You can read more about what the course covers and how it is structured [here](#the-course).
+Writing code is now an _essential part of all science and technology_ research and development - yet many of the scientists and domain experts have _never received training on software engineering_ best practices, and are instead expected to figure it out. The result of this is slow software development and unreliable software.
 
-Get in touch today to discuss getting these seminars given to you. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) or ring me on 07948 284729.
+> "Having Thomas come on site and give these seminars massively improved our productivity"
+>
+> *A Company*
 
-> "Having Thomas come on site and give these talks massively improved our productivity"
+I can deliver a course that will teach your developers the _industry standard techniques_ to write more reliable code faster.
 
-Some more text.
+Email me at [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) to discuss how this course can help your company develop software effectively.
 
 ## The Course
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+The course is designed to cover the key areas of modern software engineering best practices. Each seminar will help developers to adopt _industry standard approaches_ to developing software.
+
+The course consists of 5 seminars:
+
+ 1. [Source Control](#source-control)
+ 2. [Automated Tests](#automated-tests)
+ 3. [“Good” Code](#good-code)
+ 4. [Code Review](#code-review)
+ 5. [Continuous Integration](#continuous-integration)
+
+## About Thomas
+
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt _the importance of good software practices_ in writing correct code quickly, and in collaboration with others.
+
+In my time as team lead I was able to see how good software practices can _speed up delivery_ and increased quality.
```

Comment:
"In my time as team lead I was able to see" -> "I have seen first-hand how"

So it reads less like a CV.